makefile

```diff
@@ -38,6 +38,22 @@ CXX = $(CC)
 ##
 -include make/detect_os
 
+## FIXME(dl): move to appropriate place later
+## GPU settings
+
+ifdef STAN_GPU
```

Comment:
We need to set a STAN_GPU env variable in Jenkins. Does someone with actual access to the machine do that or is this something that is changed in the Jenkinsfile? I have to read more about Jenkins before I feel comfortable touching that file